### ISSUE TITLE
Running devtools: No module named django

### ISSUE THREAD

**Issue by ross-spencer** [artefactual/archivematica-devtools#47]
Currently following @timothyryanwalsh's instructions in METSFlask to explore the tool a little more. 

https://github.com/timothyryanwalsh/METSFlask#detailed-identification-and-dates

I get the following:

     vagrant@vagrant:~/git/artefactual/archivematica-devtools$ sudo am ensure-fits-characterization
     Traceback (most recent call last):
       File "/usr/local/libexec/archivematica/ensure-fits-characterization", line 34, in <module>
         import django
     ImportError: No module named django

AM 1.6.1. 

If I start trying to add the dependencies, e.g. pip install django, then we seem to step into a rabbits den of other dependency issues. One odd one is 'Import, no module named webdesign' which seems to be coming from the initialisation of the [django.contrib](https://docs.djangoproject.com/en/1.7/ref/contrib/) modues and should be there once django is (other contrib modules are). 

Is my expectation wrong to be able to do this from the Vagrant installation of AM?

**Comment by timothyryanwalsh** [comment]
Hi Ross. FYI, I don't think this is related to Vagrant. I ran into the same issue the other day with AM 1.6.1 on a non-Vagrant installation. The script worked fine (no ImportError) with AM 1.6.

**Comment by sevein** [comment]
@ross-spencer there is a bit of a problem with the way devtools work. It assumes that the dependencies are installed globally. In 1.6 virtualenvs were added to the Ansible role but it's not enabled by default. But it seems the Vagrant box you're using was provisioned with that option enabled. That means that the deps are installed but they're inside the environment. Each component has its own but most of the tools in this repo will work if you use the archivematica-dashboard env: /usr/share/python/archivematica-dashboard. FYI this is where the venvs are installed: https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.6.x/tasks/pipeline-pipdeps-multi-venvs.yml.

I'm not a big fan of the way devtools work. I tried to capture part of the problem here: https://github.com/artefactual/archivematica-devtools/issues/43.
